README.md

```diff
@@ -76,6 +76,11 @@ set -g @sessionx-pointer "▶ "
 # Requires zoxide installed
 set -g @sessionx-zoxide-mode 'on'
 
+# If you want to pass in your own FZF options. This is passed in before all other
+# arguments to FZF to ensure that other options like `sessionx-pointer` and 
+# `sessionx-window-height/width` still work. See `man fzf` for config options.
+set -g @sessionx-fzf-options "--color pointer:9,spinner:92,marker:46"
```

Comment:
## Not usual name

We can refactor this option to: `@sessionx-aditional-options` instead of  `@sessionx-fzf-options`
It is more readable 😄